docs/messaging/overview.mdx

```diff
@@ -57,7 +57,7 @@ public class Application extends FlutterApplication implements PluginRegistrantC
 
   @Override
   public void registerWith(PluginRegistry registry) {
-    GeneratedPluginRegistrant.registerWith(registry);
+    FirebaseMessagingPlugin.registerWith(registry.registrarFor("io.flutter.plugins.firebasemessaging.FirebaseMessagingPlugin"));
```

Comment:
Why such change? With GeneratedPluginRegistrant you shouldn't need to do that

Reply:
Yep, and even then it's `FlutterFirebaseMessagingPlugin`, FirebaseMessagingPlugin doesn't exist

Reply:
This was written in the documentation itself, I just implemented it and pushed it, nothing new. Maybe implementation changed, I will check this out.